tests/searchdocs_test.py

```diff
@@ -2,10 +2,13 @@
 
 extra_plugin_dir = 'plugins'
 
+
 def test_search_cmd(testbot):
-    testbot.assertCommand('!search api this is search string',
-                          'http://api.coala.io/en/latest/search.html?q=this+is+search+string')
-    testbot.assertCommand('!search user this is search string',
-                          'http://docs.coala.io/en/latest/search.html?q=this+is+search+string')
+    testbot.assertCommand(
+        '!search api this is search string',
+        'http://api.coala.io/en/latest/search.html?q=this+is+search+string')
+    testbot.assertCommand(
+        '!search user this is search string',
+        'http://docs.coala.io/en/latest/search.html?q=this+is+search+string')
```

Comment:
please create a newcomer or low issue for this to be changed to https

Reply:
https://github.com/coala/corobo/issues/579